packages/react-composites/src/composites/MeetingComposite/state/MeetingAdapterState.ts

```diff
@@ -71,3 +35,58 @@ export interface MeetingAdapterClientState extends Pick<CallAdapterClientState,
  * combined into one to suit the purpose of a Meeting.
  */
 export interface MeetingAdapterState extends MeetingAdapterUiState, MeetingAdapterClientState {}
+
+export function generateMeetingAdapterState(callAdapter: CallAdapter, chatAdapter: ChatAdapter): MeetingAdapterState {
+  const callAdapterState = callAdapter.getState();
+  const chatAdapterState = chatAdapter.getState();
+
+  const { call, displayName, userId, devices, page } = callAdapterState;
+  const meeting = call ? generateMeetingState(call, chatAdapterState.thread) : undefined;
+
+  return {
+    meeting,
+    userId,
+    page: callPageToMeetingPage(page),
+    displayName,
+    devices
+  };
+}
+
+export function mergeChatAdapterStateIntoMeetingAdapterState(
```

Comment:
nitty, nitty, nitttty nit: `mergeInto(dest, src)`: I'd expect `src` to be merged into `dest`. This direction is conventional and also makes for nice varargs version: `mergeInto(dest, src1, src2...)`

Reply:
This comment applies to all the `mergeInto` functions in this file and `MeetingState.ts`.

Reply:
Updated 👍